src/Dev/SapphireTest.php

```diff
@@ -1807,7 +1807,7 @@ public function getFixtureFactory()
      * Sets a new fixture factory
      * @param FixtureFactory $factory
      * @return $this
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
```

Comment:
The version in the below notice needs updating, and the message needs to be standardised
And add the message here.

Reply:
Updated